config/core/deployments/controller.yaml

```diff
@@ -52,6 +52,9 @@ spec:
           requests:
             cpu: 100m
             memory: 100Mi
+          limits:
+            cpu: 100m
+            memory: 100Mi
```

Comment:
Was this supposed to be 1000 each?

Reply:
yeah, looks like I spotted this too but the fix landed in the wrong commit - fixed in https://github.com/knative/eventing/pull/5612 :)